test/integration_test/storkctl_schedulepolicy_test.go

```diff
@@ -0,0 +1,246 @@
+//go:build integrationtest
+// +build integrationtest
+
+package integrationtest
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	"github.com/libopenstorage/stork/pkg/storkctl"
+	storkops "github.com/portworx/sched-ops/k8s/stork"
+	"github.com/portworx/sched-ops/task"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	"io"
+	"k8s.io/apimachinery/pkg/api/errors"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"os"
+	"testing"
+	"time"
+)
+
+func TestStorkCtlSchedulePolicy(t *testing.T) {
+	err := setSourceKubeConfig()
+	require.NoError(t, err, "failed to set kubeconfig to source cluster: %v", err)
+	t.Run("createDefaultIntervalSchedulePolicyTest", createDefaultIntervalSchedulePolicyTest)
+	t.Run("createDefaultDailySchedulePolicyTest", createDefaultDailySchedulePolicyTest)
+	t.Run("createDefaultWeeklySchedulePolicyTest", createDefaultWeeklySchedulePolicyTest)
+	t.Run("createDefaultMonthlySchedulePolicyTest", createDefaultMonthlySchedulePolicyTest)
+	t.Run("createCustomIntervalSchedulePolicyTest", createCustomIntervalSchedulePolicyTest)
+	t.Run("createCustomDailySchedulePolicyTest", createCustomDailySchedulePolicyTest)
+	t.Run("createCustomWeeklySchedulePolicyTest", createCustomWeeklySchedulePolicyTest)
+	t.Run("createCustomMonthlySchedulePolicyTest", createCustomMonthlySchedulePolicyTest)
+	t.Run("deleteSchedulePolicyTest", deleteSchedulePolicyTest)
+	err = setRemoteConfig("")
+	require.NoError(t, err, "setting kubeconfig to default failed")
+}
+
+func createSchedulePolicyTest(t *testing.T, policyType string, args map[string]string, specFileName string, testrailID int) {
+	var testResult = testResultFail
+	runID := testrailSetupForTest(testrailID, &testResult)
+	defer updateTestRail(&testResult, testrailID, runID)
+
+	factory := storkctl.NewFactory()
+	schedulePolicyName := "automation-test-policy"
+	var outputBuffer bytes.Buffer
+	cmd := storkctl.NewCommand(factory, os.Stdin, &outputBuffer, os.Stderr)
+	cmdArgs := []string{"create", "schedulepolicy", schedulePolicyName, "-t", policyType}
+	//add the custom args to the command if any
+	for key, value := range args {
+		cmdArgs = append(cmdArgs, "--"+key)
+		cmdArgs = append(cmdArgs, value)
+	}
+	cmd.SetArgs(cmdArgs)
+	//execute the command
+	logrus.Infof("The command being executed is %v", cmdArgs)
+	if err := cmd.Execute(); err != nil {
+		logrus.Errorf("Execute storkctl failed: %v", err)
```

Comment:
nit.. I think you borrowed this message from common_test.go but maybe we can correct there too.
I feel "Storkctl execution failed" is a better message. Same everywhere